assets/dev/scss/frontend/_modules.scss

```diff
@@ -1,2 +1 @@
 // Shapes Module
-@import '../../../../modules/shapes/assets/scss/frontend';
```

Comment:
I think now we can completely remove this file, and remove this reference from the frontend-lite.scss file as well.